docs/root-manifest.json

```diff
@@ -83,6 +83,30 @@
 		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/extensibility/autocomplete.md",
 		"parent": "extensibility"
 	},
+	{
+		"title": "Design",
+		"slug": "design",
+		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/design.md",
+		"parent": null
+	},
+	{
+		"title": "Patterns",
+		"slug": "patterns",
+		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/design\/patterns.md",
+		"parent": "design"
+	},
+	{
+		"title": "Block Design",
+		"slug": "block-design",
+		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/design\/block-design.md",
+		"parent": "design"
+	},
+	{
+		"title": "Resources",
+		"slug": "resources",
```

Comment:
We already have slug resources. I'm not sure if this won't get into conflict when processed by the script which maintains Gutenberg handbook. I would recommend the following changes to ensure it works properly on wordpress.org and we can link files in both GitHub and handbook:

File renamed to `design-resources.md`.

```json
"slug": "design-resources",
"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/design\/design-resources.md",
```